### PR TITLE
Added WAMP topic for getting balances updates for authorized users.

### PR DESCRIPTION
**Pull request by polonzer**
https://lykkex.atlassian.net/browse/LWDEV-5410
Added WAMP topic for getting balances updates for authorized users.
Moved settings classes into "Lykke.Frontend.WampHost.Core" project.

### CODE REVIEW

File: src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                            NewBalance = balance.NewBalance,
+                            NewReserved = balance.NewReserved,
+                            OldBalance = balance.OldBalance,
+                            OldReserved = balance.OldReserved
+                        } }
+                    });
+                }
+                catch (Exception ex)
+                {
+                    _log.WriteError(nameof(BalancesConsumer), message.ToJson(), ex);
```
Comment (MikhailGerasimov) [inline src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs]: Can it DDoS our error monitoring?

File: src/Lykke.Frontend.WampHost.Services/Balances/BalanceUpdateMqDeserializer.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Text;
+using Common;
+using Lykke.Frontend.WampHost.Services.Balances.IncomeMessages;
+using Lykke.RabbitMqBroker.Subscriber;
+
+namespace Lykke.Frontend.WampHost.Services.Balances
+{
+    public class BalanceUpdateMqDeserializer : IMessageDeserializer<BalanceUpdateEventModel>
+    {
+        public BalanceUpdateEventModel Deserialize(byte[] data)
```
Comment (MikhailGerasimov) [inline src/Lykke.Frontend.WampHost.Services/Balances/BalanceUpdateMqDeserializer.cs]: RabbitMq lib already includes default deserializer.

File: src/Lykke.Frontend.WampHost.Core/Settings/RabbitMqSettings.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+using Lykke.SettingsReader.Attributes;
+
+namespace Lykke.Frontend.WampHost.Core.Settings
+{
+    [UsedImplicitly]
+    public class RabbitMqSettings
+    {
+        public string ConnectionString { get; set; }
+        [Optional]
+        public string ExchangeName { get; set; }
```
Comment (MikhailGerasimov) [inline src/Lykke.Frontend.WampHost.Core/Settings/RabbitMqSettings.cs]: Exchange name can't be optional if you didn't hardcoded it somewere

Comment (polonzer) [reply src/Lykke.Frontend.WampHost.Core/Settings/RabbitMqSettings.cs]: That's true. Existing code use several hardcoded strings.

File: src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs (hunk 1 above)
Comment (polonzer) [reply src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs]: This is a normal behavior. We have a problem — we show a problem.

File: src/Lykke.Frontend.WampHost.Services/Balances/BalanceUpdateMqDeserializer.cs (hunk 2 above)
Comment (polonzer) [reply src/Lykke.Frontend.WampHost.Services/Balances/BalanceUpdateMqDeserializer.cs]: I'll try to use it.

File: src/Lykke.Frontend.WampHost.Core/Settings/RabbitMqSettings.cs (hunk 3 above)
Comment (KonstantinRyazantsev) [reply src/Lykke.Frontend.WampHost.Core/Settings/RabbitMqSettings.cs]: Why you need exchange name in the settings? It is static

File: src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            _settings = settings ?? throw new ArgumentNullException(nameof(settings));
+
+            _subject = realm.Services.GetSubject(TopicUri);
+        }
+
+        public void Start()
+        {
+            try
+            {
+                var subscriptionSettings = new RabbitMqSubscriptionSettings
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs]: Use fluent settings configuration API. Use SubscribersFactory, or at least fleunt RabbitMqSubscriptionSettings API

File: src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs (hunk 1 above)
Comment (KonstantinRyazantsev) [reply src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs]: I think, will be good to aggregate all catched in the loop exceptions to the single AggregateException and the just throw it, without logging. Infrastructure will log it. And wereduce probability of DDoS to our logging system

File: src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+            [NotNull] ILog log,
+            [NotNull] RabbitMqSettings settings,
+            [NotNull] IWampHostedRealm realm,
+            [NotNull] IClientResolver clientResolver)
+        {
+            _log = log ?? throw new ArgumentNullException(nameof(log));
+            _clientResolver = clientResolver ?? throw new ArgumentNullException(nameof(clientResolver));
+            _settings = settings ?? throw new ArgumentNullException(nameof(settings));
+
+            _subject = realm.Services.GetSubject(TopicUri);
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Frontend.WampHost.Services/Balances/BalancesConsumer.cs]: Add autodocs metadata for this topics to the ```IWampTopics```

**Pull request merged**